bugguide/bugguide
# #30 Give authenticated users "use editablefields" and give contributo…

### Description

#30 Give authenticated users "use editablefields" and give contributor roles "Image: edit any content".

### Testing steps

The scenario for " Give authenticated users "use editablefields"."

1. Log in as authenticated user
2. Go to /node/1903271 (content type: bgpage)
3. You should see button "Edit this field" in Tags section
4. You can add any tag save it.

The scenario for giving contributor role "Image: edit any content".

1. Log in as contributor
2. Go to any bgimage node pages (@todo add an example image page once this has been merged)
3. You should be able to edit the image node.
